service/common/src/main/java/org/apache/polaris/service/catalog/generic/GenericTableCatalog.java

```diff
@@ -22,163 +22,21 @@
 import java.util.Map;
 import org.apache.iceberg.catalog.Namespace;
 import org.apache.iceberg.catalog.TableIdentifier;
-import org.apache.iceberg.exceptions.AlreadyExistsException;
-import org.apache.iceberg.exceptions.NoSuchNamespaceException;
-import org.apache.iceberg.exceptions.NoSuchTableException;
-import org.apache.polaris.core.catalog.PolarisCatalogHelpers;
-import org.apache.polaris.core.context.CallContext;
-import org.apache.polaris.core.entity.CatalogEntity;
-import org.apache.polaris.core.entity.PolarisEntity;
-import org.apache.polaris.core.entity.PolarisEntitySubType;
-import org.apache.polaris.core.entity.PolarisEntityType;
 import org.apache.polaris.core.entity.table.GenericTableEntity;
-import org.apache.polaris.core.persistence.PolarisMetaStoreManager;
-import org.apache.polaris.core.persistence.PolarisResolvedPathWrapper;
-import org.apache.polaris.core.persistence.dao.entity.BaseResult;
-import org.apache.polaris.core.persistence.dao.entity.DropEntityResult;
-import org.apache.polaris.core.persistence.dao.entity.EntityResult;
-import org.apache.polaris.core.persistence.pagination.PageToken;
-import org.apache.polaris.core.persistence.resolver.PolarisResolutionManifestCatalogView;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-public class GenericTableCatalog {
-  private static final Logger LOGGER = LoggerFactory.getLogger(GenericTableCatalog.class);
+/** A catalog for managing `GenericTableEntity`s */
```

Comment:
nit you meant this ?
```suggestion
/** A catalog for managing GenericTableEntity`s */
```

Reply:
No I mean `GenericTableEntity`s or `{@link GenericTableEntity}s`... I guess if we want to be super grammatical we can say `{@link GenericTableEntity} instances`